openassessment/templates/openassessmentblock/message/oa_message_incomplete.html

```diff
@@ -1,56 +1,54 @@
 {% load i18n %}
 {% spaceless %}
 <div id="openassessment__message__{{ xblock_id }}" class="openassessment__message message">
-    <h3 class="message__title">
-        {% trans "Assignment in progress" %}
-    </h3>
+    <h4 class="message__title">{% trans "Progress" %}</h4>
```

Comment:
If there's a reason that it needs to be different from "Status", I think it makes more sense as "In Progress" than "Progress".